style.css

```diff
@@ -1,37 +1,198 @@
 * {
-  font-family: 'Comfortaa', cursive;
   margin: 0;
   padding: 0;
   box-sizing: border-box;
+  font-family: 'Comfortaa', cursive;
+  color: rgb(61, 61, 61);
 }
 
-body {
-  margin: 0;
-  padding: 1vw;
-  font-family: 'Comfortaa', cursive;
+html {
+  font-size: 62.5%;
+}
+
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 95%;
+  height: 10vh;
+  margin-top: 1rem;
+  border: 2px solid black;
+}
+
+.logo {
+  display: flex;
+  gap: 1rem;
+}
+
+.logo-icon {
+  margin-left: 1rem;
+  font-size: 1.7rem;
+}
+
+.logo-title {
+  font-size: 1.7rem;
+}
+
+.nav-list {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  list-style: none;
+}
+
+.nav-item {
+  padding-left: 1rem;
+  padding-right: 1rem;
+  border-right: 1px solid black;
+  font-size: 1.7rem;
+  cursor: pointer;
+}
+
+.nav-item:last-child {
+  border-right: none;
+}
+
+.hero {
+  width: 95%;
+  height: 87vh;
+  padding-top: 3rem;
+}
```

Comment:
- [ ] You still have not resolved the footer issue as highlighted by the previous reviewer (see image below). All through your CSS you are using **`vh`** unit for your heights. But technically, the total viewport-height is 100. When all the **`vh`** values you have are added the result is more than 100.

  Here is a suggestion: Remove the line **`height: 87vh;`** here. It should correct your styling issue.

![oliver-foot](https://user-images.githubusercontent.com/36542986/144627523-4cbfe611-aa74-4223-9ae2-54ade9059ebf.jpg)